src/components/navigation-bar/navigation-bar.component.js

```diff
@@ -1,29 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import propTypes from '@styled-system/prop-types';
 import AppWrapper from '../app-wrapper/app-wrapper';
 import StyledNavigationBar from './navigation-bar.style';
 
 const NavigationBar = ({
   navigationType = 'light',
   isLoading = false,
   children,
-  ariaLabel
+  ariaLabel,
+  ...props
 }) => {
+  const spacingProps = {};
+
+  Object.keys(props).forEach((propName) => {
```

Comment:
See my comment below.
If we choose to keep these props separate I think we can convey the intention better by writing this as a reducer.

```js
const spacingProps = Object.keys(props).reduce((acc, propName) => {
      acc[propName] = props[propName];
      return acc;
    }, {});
```

Reply:
Yeah I think separating them is worthwhile for the above reason, but using a reducer does convey it a little better so I'll update it